db/schema.rb

```diff
@@ -10,32 +10,32 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170115230058) do
+ActiveRecord::Schema.define(version: 20170123173604) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "hstore"
 
   create_table "activities", force: :cascade do |t|
     t.integer  "team_id"
-    t.string   "kind",         limit: 255
-    t.string   "guid",         limit: 255
-    t.string   "author",       limit: 255
-    t.string   "title",        limit: 255
+    t.string   "kind"
```

Comment:
Yes, it relies on the used psql version. Because of that I am always in favour of using only one postgres version everywhere. From dev over staging to production. Even small version differences can have huge changes in behavior. I have a tool called [pgvm](https://github.com/guedes/pgvm) on my machine when switching between projects. Other people I know use separate docker containers for each project and only work inside of the container.